css/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #191765;
+  font-family: Roboto, sans-serif;
+  background-image: url("../images/Top\ circles.svg");
+  z-index: 99;
+  background-repeat: no-repeat;
+  background-position: center top;
+  background-size: cover;
+  padding-top: 50px;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+  align-items: center;
+}
+
+header a {
+  color: #fff;
+  height: 16px;
+  width: 200px;
+  font-size: 10px;
+  top: 46px;
+  display: flex;
+  align-items: center;
+}
+
+header nav img {
+  width: 50px;
+  height: 66px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 24px;
+}
+
+section h1 {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  font-size: 48px;
+  color: #7d73fc;
+  line-height: 72px;
+  font-weight: 700px;
+  text-align: center;
+  letter-spacing: 0.37px;
+}
+
+section span {
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #edecff;
+}
+
+section p {
+  padding-top: 30px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  width: 327px;
+  color: #c8c8d4;
+  text-align: center;
+  padding-bottom: 20px;
+  margin: 9px auto;
+}
+
+.breakage {
+  width: 100%;
+  padding: 24px;
+}
+
+.skillset {
+  display: flex;
+  flex-wrap: column wrap;
+  padding-top: 20px;
+  justify-content: space-evenly;
+  align-items: stretch;
+  line-height: 24px;
+  gap: 100px;
+}
```

Comment:
\[OPTIONAL\]
- [ ] It is important to follow the Figma design closely. You have really done great so far but there's something little you are missing. On smaller devices, your skillset goes away from the design. The following screenshot was taken on a screen size of `443px`. I suggest you change your flexbox properties on small screens so it looks as close as the Figma design.

![image](https://user-images.githubusercontent.com/29071984/170609621-0b7c48d5-e4b9-4981-ab01-22d71f9b1d0a.png)

This is the Figma design
![image](https://user-images.githubusercontent.com/29071984/170610380-5bddd52e-4a5c-4b9c-81ea-b3f820c30f1b.png)

Reply:
@cwaku , Thanks so much for your declarative reviews. will revert this change and merge. thank you.